runners.go

```diff
@@ -375,20 +375,33 @@ func (s *RunnersService) ListGroupsRunners(gid interface{}, opt *ListGroupsRunne
 	return rs, resp, err
 }
 
+// RegisterNewRunnerInfo represents the info hashmap parameter in
+// RegisterNewRunnerOptions.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/runners.html#register-a-new-runner
+type RegisterNewRunnerInfo struct {
```

Comment:
All option struct end with the word `Options`, so this one should too. Not sure what the best name would be in this case, but maybe something like `NewRunnerMetadataOptions`? Of `NewRunnerInfoOptions`?